regression-tests/backends/geoip-master

```diff
@@ -74,6 +80,12 @@ EOF
 2	.	IN	OPT	0	AAgACAABIBgBAgME
 Rcode: 0 (No Error), RD: 0, QR: 1, TC: 0, AA: 1, opcode: 0
 Reply to question for qname='continent.geo.example.com.', qtype=TXT
+EOF
+		cat > $testsdir/custom-mapping-a-resolution/expected_result <<EOF
+0	map.geo.example.com.	IN	A	30	$geoipregionip
+2	.	IN	OPT	0	AAgACAABIBgBAQEB
+Rcode: 0 (No Error), RD: 0, QR: 1, TC: 0, AA: 1, opcode: 0
+Reply to question for qname='map.geo.example.com.', qtype=A
 EOF
```

Comment:
See my comment about the A value of `earth.map.geo.example.com.`

Reply:
Created file with TXT value.